rules/azure-resources-creating/rule.md

```diff
@@ -21,14 +21,13 @@ created: 2020-10-06T00:13:27.000Z
 archivedreason: null
 guid: 007dd1f6-8ac6-4840-8f4f-a39c6f847880
 ---
-
 We've been down this road before where developers had to be taught not to manually create databases and tables. Now, in the cloud world, we're saying the same thing again:  **Don't manually create Azure resources.**
 
 <!--endintro-->
 
 ### Manually Creating Resources
 
-This is the most common and the worst. This is bad because it requires manual effort to reproduce and leaves margin for human error.
+This is the most common and the worst. This is bad because it requires manual effort to reproduce and leaves margin for human error. Manually provisioning resources can also lead to [configuration ddrift](https://learn.microsoft.com/en-us/devops/deliver/what-is-infrastructure-as-code#avoid-manual-configuration-to-enforce-consistency), which is to say that over time it can be difficult to keep track of which deployment configurations were made and why.
```

Comment:
```suggestion
This is the most common and the worst. This is bad because it requires manual effort to reproduce and leaves margin for human error. Manually provisioning resources can also lead to [configuration drift](https://learn.microsoft.com/en-us/devops/deliver/what-is-infrastructure-as-code#avoid-manual-configuration-to-enforce-consistency), which is to say that over time it can be difficult to keep track of which deployment configurations were made and why.
```

Reply:
Done - I committed the change